Dockerfile

```diff
@@ -1,14 +1,23 @@
 ARG repository=rapidsai-dev-nightly
-ARG version=0.11-cuda10.0-devel-ubuntu18.04-py3.7
+ARG version=0.12-cuda10.0-devel-ubuntu18.04-py3.7
 
 FROM rapidsai/${repository}:${version}
 
-ADD . /clx/
+ADD . /rapids/clx/
 
 SHELL ["/bin/bash", "-c"]
+
 RUN source activate rapids \
-    && cd /clx \
+    && conda install panel=0.6.* geopandas pyppeteer \
```

Comment:
I'd recommend adding `--freeze-installed` just incase a new version of one of these packages tries to force updating a rapids dependency outside of our supported versions.

Reply:
added `--freeze-installed`